client/dashboard/sites/invite-accepted-flash-message.tsx

```diff
@@ -0,0 +1,74 @@
+import { useDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useEffect } from 'react';
+
+export function InviteAcceptedFlashMessage() {
```

Comment:
I copy pasted the standard `<FlashMessage>` component rather than try and bake this special logic into the standard component. Was that the right thing to do? On the one hand I don't want to complicate the component, on the other hand this is all duplication, and maybe this is something we will want to do in the future too?

The only way I can think of making `<FlashMessage>` handle something like this would be to allow the `message` prop to be a callback that allowed arbitrary code to run and then return a string message.